.github/workflows/build-and-test.yml

```diff
@@ -33,6 +33,7 @@ jobs:
             6.x
             7.x
             8.x
+            9.x
```

Comment:
It it a bit weird that you install all these framework, to not have them in a matrix.

Reply:
I think that for this specific workflow, it makes more sense to have all the used .NET versions available at the same runner process. Other than this, it would trigger too many jobs. Maybe we can test this approach in another PR, what do you think?

Reply:
If you care to support the combination of all these dotnet versions on all these operating systems, then a matrix is the way to go.
I wouldn't worry about too many jobs as these will run in parallel anyway.

Reply:
Indeed, it may be better to get specific problems. I'll open a PR testing this setup later today

Reply:
https://github.com/giraffe-fsharp/Giraffe/pull/644